.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -1,7 +1,7 @@
 name: 🐞 Bug
 description: Report a bug or an issue you've found within the dbt package
 title: "[Bug] <title>"
-labels: ["bug", "triage"]
+labels: ["type:bug", "triage"]
```

Comment:
Suggested change to the template to be in line with out current labeling. Additionally, I think we can remove the triage label since it doesn't exist in our environments.

Reply:
Speaking of which. Can you work with Sheri to understand how to add the new labels to this (and the transform) repo.

Reply:
added!